### ISSUE TITLE
Make texture merging for child nodes

### ISSUE THREAD

**Issue by yeluoyunfei** [openglobus/openglobus#470]
when  zoom out from ready tiles, get images from child nodes

**Comment by program-sam** [comment]
Yes I still believe this would be a great feature! 👍 Even just temporarily until the correct texture is loaded. Especially when the globe starts zoomed in, it currently looks very patchy when zooming out because just nothing shows.

**Comment by Zemledelec** [comment]
dupe https://github.com/openglobus/openglobus/issues/796
